app/submitter/converter.py

```diff
@@ -192,7 +193,7 @@ def _encode_value(value):
     if isinstance(value, int):
         return str(value)
     elif isinstance(value, datetime):
-        return value.strftime(settings.SDX_DATE_FORMAT)
+        return value.strftime(SDX_DATE_FORMAT)
```

Comment:
@jonnyshaw89 Seems we don't have a test which covers it either! I can remove it, do we just store dates as string values now?